src/reactapp/src/pages/Main.js

```diff
@@ -29,12 +29,18 @@ export default function EvaluationTable() {
 
 
   useEffect(() => {
-    fetchEvaluationRatings();
+    const userId = getLoggedInUserId();
+
+    if (userId == null) {
+      window.location.href = "/login";
+    }
+
+    fetchEvaluationRatings(userId);
   }, []);
 
   //const response = await get("/connect/token", tokenRequest);
 // Retrieve evaluation ratings from API
-  const fetchEvaluationRatings = async () => {
+  const fetchEvaluationRatings = async (userId) => {
     try {
       const userId = getLoggedInUserId();
```

Comment:
```suggestion
      let response;

      if (getLoggedInUserRole() === 'Supervisor') {
        response = await get('/api/EvaluationRating/');
      } else {
        response = await get(`/api/EvaluationRating/${userId}`);
      }
```
@DavidJGapCR can you replace lines 45-47 with the above? We don't need the extra call to getLoggedInUserId() and when the user is a supervisor they get all evaluations. I just noticed this was missing today, this will save me a ticket and a PR